src/index.js

```diff
@@ -17,7 +18,9 @@ ReactDOM.render(
       <ProductFilterProvider>
```

Comment:
suggestion! u can use one provider which  contains all other providers

Reply:
Wow. That's a nice idea. never thought about it. Thanks man